pcweb/pages/docs/hosting/self_hosting.py

```diff
@@ -104,7 +104,7 @@ def self_hosting():
         subheader("Pynecone Container Service"),
         doctext(
             "Another option is to run your Pynecone service in a container.",
```

Comment:
It is better to add the space in this line after the ```... container. "```
This is the way how the parser works.

Reply:
Should I create a new pull request with the correction handled the way you suggested? Thanks!

Reply:
No, you can fix it, commit and push it in the same branch. That will add it automatically to the PR here. :)

Reply:
Awesome, will do, thanks!